src/domain/usecases/user-monitoring/GetLogFormatDate.ts

```diff
@@ -0,0 +1,13 @@
+export class GetLogFormatDate {
+    private logFormatDate(date: Date): string {
+        const isoString = date.toISOString();
+
+        return isoString.replace("Z", "");
+    }
+
+    constructor() {}
+
+    execute(date: Date): string {
+        return this.logFormatDate(date);
+    }
+}
```

Comment:
it can be a simple function. Or we can keep a class, with a more declarative for "execute", in case we expect to add more methods to this Date wrapper.

Reply:
Changed to a function.